### ISSUE TITLE
Data structure: Date missing values not replaced

### ISSUE THREAD

**Issue by geofranzi** [BEXIS2/Core#641]
- [x] Primary data view
- [x] Export

![image](https://user-images.githubusercontent.com/15946467/91988454-92c74480-ed2f-11ea-898c-5931d3e9a760.png)
![image](https://user-images.githubusercontent.com/15946467/91988516-a4105100-ed2f-11ea-8eac-b1dc9ee2ec58.png)

it looks for the full default string, but it is already converted to year.

![image](https://user-images.githubusercontent.com/15946467/91989047-547e5500-ed30-11ea-88a0-45061f18a76b.png)

**Comment by geofranzi** [comment]
ShowPrimaryDataModel.cs L121 change to 

```
 foreach (var missingValue in variable.MissingValues)
                    {

                        if (DataTypeUtility.GetTypeCode(variable.DataAttribute.DataType.SystemType) == DataTypeCode.DateTime)
                        {
                            DataTypeDisplayPattern ddp = DataTypeDisplayPattern.Materialize(variable.DataAttribute.DataType.Extra);
                            DateTime dateTime;
                            if (DateTime.TryParse(missingValue.Placeholder, new CultureInfo("en-US", false), DateTimeStyles.NoCurrentDateDefault, out dateTime))
                            {
                                sb.Append(missingValue.DisplayName + "|" + dateTime.ToString(ddp.StringPattern) + "#"); ;
                            }

                        }
                        else
                        {
                            sb.Append(missingValue.DisplayName + "|" + missingValue.Placeholder + "#");
                        }
                    }
```

**Comment by geofranzi** [comment]
The current solution does not work for time e.g.  11:59 will be also replaced.

In theory, **placeholders** for missing values should be **NOT formatted**. Formatting is currently done by Telerik.

**Comment by geofranzi** [comment]
Formatting before does not work, as it does not fit with the datatype of the DataTables column. Currently no idea how to solve it. If we change the DataType I assume filtering will not work. Maybe stay with my 1st solution until we may change to datatables instead of Telerik.

**Comment by geofranzi** [comment]
- [ ] fix export in Milestone 1.14.3
